### PR TITLE
Streamlining, resolves #79

### LINKED ISSUE #79: modularize commands
this is the first step in making our bot platform-agnostic

### PR DESCRIPTION
**Pull request by naterchrdsn**

### CODE REVIEW

File: lib/commands.js [hunk 1]
```diff
@@ -1,20 +1,19 @@
 const Kirbi = require('../kirbi');
```
Comment (Morandustries) [inline lib/commands.js:1]: looks like this file has spaces instead of tabs

File: lib/commands.js [hunk 2]
```diff
@@ -60,5 +55,29 @@ exports.init = function() {
...
+            if (Kirbi.Commands[module]) {return};
+            //grab from node_modules
+            var dir = '/node_modules/kirbi-' + module;
+                try {
+                    module = Kirbi.require(`${path.join(dir, module)}`);
+                } catch (err) {
+                    Kirbi.logError(`Improper setup of the '${module}' command file. : ${err}`);
+                    return;
+                }
+                if (module) {
```
Comment (Morandustries) [inline lib/commands.js]: just add && module.commands and remove the extra nesting below

File: lib/onEvent/message.js [hunk 3]
```diff
@@ -6,9 +6,19 @@ if (Kirbi.Config.elizaEnabled) {
...
-}
+};
 
-function checkMessageForCommand(msg, isEdit) {
+function discordMessageCb (output, msg, expires, delCalling) {
+	if (expires) {
+		return msg.channel.send(output).then(message => message.delete(5000));
+	}
+	if (delCalling) {
+		return msg.channel.send(output).then(message => msg.delete());
```
Comment (Morandustries) [inline lib/onEvent/message.js]: remove message from the then since its not used. replace with `()`

**Review by Morandustries — CHANGES_REQUESTED** [review]
Couple comments

File: lib/onEvent/message.js (hunk 3 above)
Comment (naterchrdsn) [reply lib/onEvent/message.js]: fixed

File: lib/commands.js (hunk 2 above)
Comment (naterchrdsn) [reply lib/commands.js]: fixed

**Review by Morandustries — APPROVED** [review]

**Pull request merged**